src/components/shared/footer/Footer.module.scss

```diff
@@ -0,0 +1,26 @@
+@use '@variables' as var;
+@use '@mixins' as mixins;
+
+.container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+  height: 378px;
+  background-color: var.$lightGrey;
+  @include mixins.pagePadding;
+}
+
+.footerNav {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+  max-width: 1131px;
```

Comment:
This 1131px is a common value that is being used as a wrapper. Can we add it to the pagePadding mixin? That way every component that uses the wrapper will have the same max width and padding.

Reply:
I went ahead and created variables for max-widths in the variables.module.scss in the rem-mixin pr. Theres an issue if you add a max width including padding on the same container. It makes the child elements padded smaller than it should be.